libs/cardano-ledger-test/bench/Bench/Cardano/Ledger/ApplyTx.hs

```diff
@@ -97,8 +96,11 @@ benchApplyTx px =
   benchWithGenState px pure $ \applyTxEnv ->
     let ApplyTxEnv {ateGlobals, ateMempoolEnv, ateState, ateTx} = applyTxEnv
      in nf
-          ( either (error . show) id
-              . applyTxsTransition ateGlobals ateMempoolEnv (Seq.singleton ateTx)
+          ( \st ->
+              either
+                (error . show)
+                id
+                (fst <$> applyTx ateGlobals ateMempoolEnv st ateTx)
```

Comment:
You don't need to change it, but I though I'd point it out anyways :wink: 
```suggestion
                fst
                (applyTx ateGlobals ateMempoolEnv st ateTx)
```

Reply:
Thanks!